adcc/modified_transition_moments.py

```diff
@@ -72,10 +74,39 @@ def compute_modified_transition_moments(gs_or_matrix, dipole_operator,
                         "libadcc.OneParticleOperator.")
 
     if method.is_core_valence_separated:
-        raise NotImplementedError("compute_modified_transition_moments "
-                                  "not implemented for CVS")
-
+        return modified_transition_moments_cvs(gs_or_matrix,
+                                               dipole_operator, method)
     mtm_cpp = libadcc.compute_modified_transition_moments(
         method.name, ground_state, dipole_operator
     )
     return AmplitudeVector(*mtm_cpp.to_tuple())
+
+
+def modified_transition_moments_cvs(gs_or_matrix, dipole_operator,
+                                    method=None):
+    if isinstance(gs_or_matrix, AdcMatrix):
+        ground_state = gs_or_matrix.ground_state
+        intermediates = gs_or_matrix.intermediates
+    elif isinstance(gs_or_matrix, LazyMp):
+        ground_state = gs_or_matrix
+        intermediates = AdcMatrix(method, ground_state).intermediates
+
+    if method.name == "cvs-adc1":
+        return AmplitudeVector(dipole_operator['o2v1'])
```

Comment:
I think I would do an `elif` for `"cvs-adc2"` and bail out with a `NotImplementedError` in an `else` block.

Reply:
Plus, I'd move the first `if`-statement for CVS-ADC(1) to the top and move the construction of intermediates etc. inside the `elif` block for CVS-ADC(2).